tests/VirtoCommerce.CustomerReviews.Test/VirtoCommerce.CustomerReviews.Test.csproj

```diff
@@ -4,7 +4,7 @@
     <noWarn>1591</noWarn>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="coverlet.collector" Version="3.1.0">
+    <PackageReference Include="coverlet.collector" Version="3.1.1">

```

Comment:
Why did you up this dependency?

Reply:
Good question. I don't know. Maybe VS 2022 did it automatically. Should I revert it back